src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1147,9 +1147,12 @@ private void NextScheduledActivityCallback(string databaseName, IdleDatabaseActi
                     }
                 }
             }
-            catch
+            catch(Exception e)
             {
                 // we have to swallow any exception here.
+
+                if (_logger.IsInfoEnabled)
```

Comment:
This isn't expected, so let's put it in the `Operations` logs

Reply:
Done.